src/crocoddyl_motion_server_node.cpp

```diff
@@ -125,8 +125,8 @@ namespace panda_torque_mpc
             std::string urdf_path = ros::package::getPath("panda_torque_mpc") + "/urdf/robot.urdf";
 
             // Building the GeometryModel
-            auto collision_model = boost::make_shared<pinocchio::GeometryModel>();
-            collision_model = loadPandaGeometryModel(model_pin_);
+            auto collision_model_ = boost::make_shared<pinocchio::GeometryModel>();
+            collision_model_ = loadPandaGeometryModel(model_pin_);
```

Comment:
```suggestion
            auto collision_model = loadPandaGeometryModel(model_pin_);
```
Same as before. This is not a class member. Plus, this object is first initialized in your case with empty object to later be overwritten. The same result will be with this code.